discovery/mdns.md

```diff
@@ -14,12 +14,11 @@ As responses come in, the peer adds the other peers information into is local da
 
 `service-name` is the DNS-SD service name for all peers. It is defined as `_p2p._udp.local`.
 
-`host-name` is the name of the peer.  It is derived from the peer's ID and `p2p.local`, for example 
-`Qmid.p2p.local`.
+`host-name` is the fully qualified name of the peer.  It is derived from the peer's name and `p2p.local`.
 
-`peer-id` is the ID of the peer.  It normally is the base-58 enconding of the hash of the peer's public key.
+`peer-name` is the case-insenstive ID of the peer and less than 64 characters.  It is normally is the base-32 encoding of peer's ID.

```

Comment:
I guess until https://github.com/ipfs/go-ipfs/issues/5287 is solved this would this require a manual step of converting back to Base58, yes?

Reply:
> I guess until ipfs/go-ipfs#5287 is solved this would require apps to do a manual step of converting back to Base58, yes?

`peer-name` is just a unique name, we should not assume it's the peer id. The `dnsaddr` ends with the peer ID.

> What if PeerID is longer than 64 characters? Should we split after 63 characters like noted in ipfs/in-web-browsers#89 ?

I think **Workaround A: Split at 63rd character** makes sense.